addons/mail/models/discuss/ir_attachment.py

```diff
@@ -1,11 +1,13 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import api, models
+from odoo import api, models, fields
 
 
 class IrAttachment(models.Model):
     _inherit = "ir.attachment"
 
+    voice_ids = fields.One2many("discuss.voice.metadata", "attachment_id", store=False)
```

Comment:
to be investigated separately why store would have an effect on query count (if any), and possibly improve orm?